.github/linters.yml

```diff
@@ -0,0 +1,45 @@
+name: Linters
```

Comment:
- [ ] There are some linter configurations that are missing in the root folder. 

- (files missing)

![image](https://user-images.githubusercontent.com/109441277/235517639-7aac4759-8340-4e66-a5ef-1044df3bfee8.png)

It would be nice if you have those so other team members (if you're working in a team) can understand better your code and you can have a more standarized way of coding. You can find the linter's configuration for this project [here](https://github.com/microverseinc/linters-config/tree/master/react-redux).